lib/src/components/NxToast/NxToast.tsx

```diff
@@ -38,7 +38,8 @@ const NxToast = (props: NxToastProps) => {
   };
 
   const validChild = React.Children.only(children),
-      childrenWithProps = React.cloneElement(validChild, { onClose: handleClose });
+      childrenWithProps = React.cloneElement(validChild,
+          { onClose: handleClose, role: children.props.role ? children.props.role : 'alert' });
```

Comment:
```suggestion
          { onClose: handleClose, role: children.props.role ?? 'alert' });
```

Also the more typical formatting would be to break the object declaration into multiple lines like `classes` below

Reply:
> Also the more typical formatting would be to break the object declaration into multiple lines like `classes` below

Broken into multiple lines here: [3e74381](https://github.com/sonatype/sonatype-react-shared-components/pull/766/commits/3e743819fa73a174bc9e3ebce8a624cd610f9ea8)